atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/ManyClientPoolingContainer.java

```diff
@@ -134,7 +134,7 @@ public synchronized void shutdownPooling() {
 
     private PoolingContainer<Client> delegateForHost(InetAddress host) {
         for (Map.Entry<InetSocketAddress, PoolingContainer<Client>> entry : containerMap.entrySet()) {
-            if (entry.getKey().equals(host)) {
+            if (entry.getKey().getAddress().equals(host)) {
```

Comment:
Does this need to be null-safe (`java.net.InetSocketAddress#getAddress` returns null if address is unresolved, but I think by this point it will have been resolved)?